string[] classes
float32[] distance 
int64[] x_center
int64[] y_center
int16[] id